### ISSUE TITLE
The page has been suspended

### ISSUE THREAD

**Issue by Mitchellbrine** [JamOORev/Website#58]
This is probably either a minor or REALLY major issue. Adding people to this to make sure it gets noticed. Like RIGHT AWAY.

**Comment by Mitchellbrine** [comment]
@hawks008 @CandiceJoy @allout58 @Glassmaker 

I want to draw your attention to this

**Comment by allout58** [comment]
Pics?

**Comment by Glassmaker** [comment]
@Mitchellbrine, both production and dev website servers are working fine at the moment. o.o

**Comment by Mitchellbrine** [comment]
It's appearing again...

**Comment by Mitchellbrine** [comment]
BUT I think it has to do with being close to an end of month period. So it might have to do with a host service being renewed (I don't know).

**Comment by KJ4IPS** [comment]
@candice Go pay your dreamhost bills!

**Comment by KJ4IPS** [comment]
Oops, meant @CandiceJoy, sorry for the false notification @candice

**Comment by Mitchellbrine** [comment]
@KJ4IPS She might be on a monthly payment plan, so we'll see. The issue resolved itself after ~3 days last time.

**Comment by Mitchellbrine** [comment]
Can confirm, we need @theminecoder's help! HALP!

**Comment by KJ4IPS** [comment]
And it is now no longer resolving. In about a week or two it'll hit the registrars. I may just snap it up and host as a service to jamoorev.

**Comment by KJ4IPS** [comment]
Pinging @Kris1432

**Comment by Kris1432** [comment]
i'm not part of the JamOORev KJ

**Comment by KJ4IPS** [comment]
I know, but I thought you may have owned the domain...

**Comment by Kris1432** [comment]
nope, Hawks does.

**Comment by Mitchellbrine** [comment]
Secondary ping @hawks008

**Comment by Mitchellbrine** [comment]
I mean @theminecoder

**Comment by KJ4IPS** [comment]
Well, I now own the domain, and will be starting hosting out of my [fork](https://github.com/Jam-Craft/JamOORev-Website) within the week. PR's here will be manually merged (if appropriate and possible).

**Comment by CandiceJoy** [comment]
Yeah, because it's not important for the owner of the community to have the domain or anything.  I was waiting for it to expire to grab it.  I banned you for a reason, now leave us alone.

**Comment by KJ4IPS** [comment]
If you like to set the dripping sarcasm aside, we can resolve this as
professionals.

If you want any DNS entries made, poke me and I'll ad them as long as they
are not illegal or morally objectionable.

If you really want the domain, I can sell it to you for my cost of
registration (once the 30 day hold period is done)

**Comment by Kris1432** [comment]
You are free to keep that domain @KJ4IPS as we will not be purchasing it from you. 

I am unsure as to why you purchased the domain as you were clearly banned from the community and you have no right to own that domain name, the polite thing to do would have been to notify us of its availability.

**Comment by KJ4IPS** [comment]
If I had waited, it would have been nabbed by a squater. Usually they reg
them for a month (ICANN's minimum) once the delete lists go around (usualy
1-2 days after deletion) and extort as much as they can... I really don't
care. However, you and Candice have not (nor never had) the right to remove
me from JamCraft, you took the sweat from my brow, and offered nothing but
petulence-filled excuese when asked. No protocol was followed, those with
power acted. The official rules for handling member disconduct banning were
ignored. No-one asked for my resignation, which given the circumstances, I
likley would have given it.
